src/components/Modal/ConfirmModal/index.tsx

```diff
@@ -61,28 +87,27 @@ export const ConfirmModal = ({
   bodyScrollLockTargetId,
   delegateCloseControl = false,
   testId,
-}) => {
+}: Props) => {
   const { confirmColor, confirmText } = defaultTypeProps(type);
   const { modalId, bodyScrollLockTarget } = useScrollLock(
     bodyScrollLockTargetId,
     modalType
   );
 
-  const _onOk = (onOk) => {
+  const _onOk = () => {
     enableBodyScrollLock(bodyScrollLockTarget, delegateCloseControl);
     onOk();
     return Promise.resolve();
   };
 
-  const _onClose = (onCloseAction) => {
+  const _onClose = () => {
     enableBodyScrollLock(bodyScrollLockTarget, delegateCloseControl);
-    onCloseAction && onCloseAction();
+    onCloseAction?.();
```

Comment:
```suggestion
    onCloseAction();
```
default 값 들어가있어서 빠져도 될것같습니다!